### ISSUE TITLE
Add E3SM v2 LE metadata

### ISSUE THREAD

**Issue by chengzhuzhang** [E3SM-Project/CMIP6-Metadata#22]
Add metadata definition files for 16 historical ensemble and 21 ssp 370 ensembles.

**Comment by chengzhuzhang** [comment]
Hey @TonyB9000 could you handle this? The can based on the v2 historical runs, and v1 LE ssp370 as examples..and it might be more straightforward to one historical and one ssp370 templates and then fill in the ens/branch time values.

**Comment by TonyB9000** [comment]
Yes, use the v1_LE historical metadata to model the v2_LE, and likewise the v1_LE ssp370 to model the v2_LE ssp370, cognizant of the "branch-time-in-parent" values.

**Comment by chengzhuzhang** [comment]
> (I'll need someone to check my work ...)
>
that would be me..:)

**Comment by chengzhuzhang** [comment]
closed by #26
